rsqueakvm/model/compiled_methods.py

```diff
@@ -380,26 +408,32 @@ def as_string(self, markBytecode=0):
 
     def guess_containing_classname(self):
         w_class = self.compiled_in()
-        if w_class and w_class.has_space():
+        if w_class:
```

Comment:
Maybe start explicitly with `if w_class is None:` and switch the return statements.